src/cli/README.md

```diff
@@ -807,6 +808,19 @@ Delete all stored settings, and signal a platform reset.
 > factoryreset
 ```
 
+### fake
+
+Send fake Thread messages.
+
+Note: Only for certification test.
+
+#### fake /a/an \<dst-ipaddr\> \<target\> \<meshLocalIid\>
```

Comment:
It seems a bit weird to me as the name `meshLocalIid` seems representing an interface identifier, however below `::1111:2222:3333:4444` is an IPv6 address format.

Reply:
Yes, it's a bit wired but currently there is no method for parsing an IID.

I will use the similar solution of `bbr mgmt dua` to parse hex string (e.g. 1111222233334444) to IID using `Hex2Bin`.